docs/send-data/opentelemetry-collector/remote-management/st-with-secrets.md

```diff
@@ -0,0 +1,96 @@
+---
+id: st-with-secrets
+title: Setting Environment variable with secret values for Source Template
+sidebar_label: Setting env variables
+description: Steps for setting environment variable with secret value which can be used by source template at runtime in a remotely managed opentelemetry collector.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+Certain source template use receiver which use secret/sensitive values like password to pull the data from service and send it to Sumo Logic using OpenTelemetry collector. For example [PostgreSQL receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/postgresqlreceiver) uses PostgreSQL username and password to bring in metrics from PostgreSQL instance. 
+
+Below are the steps to make things work for such source templates : 
+
+1. Set env variable in OpenTelemetry Collector process. For example : SECRET_ENV_VAR=Welcome@123 . Where SECRET_ENV_VAR is the variable name and Welcome@123 is password.
+2. ST creation : While creating the source template you will need to provide the environment variable name instead of the actual password. In this case : SECRET_ENV_VAR and this will be part of the source template referring to the value of the environment variable like: 
+`password: ${env:SECRET_ENV_VAR}`
```

Comment:
```suggestion
2. ST creation: While creating the source template you will need to provide the environment variable name instead of the actual password. In this case: `SECRET_ENV_VAR` and this will be part of the source template referring to the value of the environment variable like: `password: ${env:SECRET_ENV_VAR}`.
```